web-app/src/app.service.ts

```diff
@@ -28,7 +28,7 @@ export class AppService {
     private gamesRepository: Repository<Game>,
     @InjectRepository(User)
     private usersRepository: Repository<User>,
-  ) {}
+  ) { }
```

Comment:
You should avoid commiting unmodified files like this, to avoid unnecessary merge conflicts 😉